### ISSUE TITLE
Split `difficulty/newcomer` label into `difficulty/trivial` and `newcomer` labels

### ISSUE THREAD

**Issue by Makman2** [coala/meta#115]
Just because an issue is trivial, it doesn't mean it's suited for newcomers.

**Comment by palash25** [comment]
So how would the newcomer process change after this.
I'm guessing they can still just do one `newcomer` issue and then move on to `trivial` and `low` issue ?
Can `trivial` be equivalent to `good-second-issue`?

**Comment by Makman2** [comment]
No `newcomer` wouldn't indicate a difficulty-level then, but just marks issues suitable for newcomers. This is now totally decoupled from the difficulty-levels. Usually all `difficulty/trivial` issues would be tagged with `newcomers`, but we can apply the label on other issues as well or not applying it at all, even for easier issues.

**Comment by jayvdb** [comment]
I do like the term 'trivial' as a difficulty level.

'newcomer'-friendly is orthogonal to difficulty levels, but we tend to not get difficulty levels right either.

I dont think it makes sense to change our current system until we have a better methodology for choosing trivial, low, medium difficulty (at the moment is only slightly better than completely random for newcomer vs low), and describe what newcomer actually is, and when we would use it, if at all.

I suspect that we can clearly define what is trivial and consistently apply that label, which will give 'low' some extra clarity.

I doubt we'll be able to do that with 'newcomer', especially not while that term is tied to an onboarding process.

So it might be easier to deprecate newcomer entirely, relabel most of them to new label trivial, and then go through the trivial issues and relabel some to `low` when they dont meet the definition of trivial.

**Comment by Makman2** [comment]
> 'newcomer'-friendly is orthogonal to difficulty levels, but we tend to not get difficulty levels right either.

Not necessarily, I (and some others) wanted to not mark documentation issues as newcomer-friendly as soon as writing new documentation is involved (even if it's trivial to just add 1-2 sentences).

> but we tend to not get difficulty levels right either.

Yeah this is always a problem. However I think we aren't doing it *that* bad :) Can still be improved though :D

> So it might be easier to deprecate newcomer entirely, relabel most of them to new label trivial, and then go through the trivial issues and relabel some to low when they dont meet the definition of trivial.

So you want to align `difficulty/trivial` to the definition of newcomer-suitability?
